cli/dcoscli/test/package.py

```diff
@@ -32,6 +34,12 @@ def package(package_name, deploy=False, args=[]):
         assert returncode == 0
         watch_all_deployments()
 
+        @retrying.retry(wait_fixed=1000, stop_max_attempt_number=300)
+        def wait_for_app_count():
+            assert count_apps() == nb_apps_before_install
```

Comment:
The reason you don't wait for the specific app to no longer be there is because of the possible disconnect between the package name and the app name, right?